ui/app/components/log-stream/index.ts

```diff
@@ -21,6 +26,22 @@ export default class LogStream extends Component<LogStreamArgs> {
 
   addLine(line: string) {
     this.lines = [...this.lines, line];
+    if (this.isFollowingLogs === false) {
+      this.badgeCount = this.badgeCount + 1;
+    }
+  }
+
+  @action
+  followLogs(element: any) {
+    element.target.parentElement.scroll(0, element.target.parentElement.scrollHeight);
```

Comment:
This is ok, but brittle if we change the DOM structure, would `this.element.querySelector('.output-scroll-y')` be better?